elements_imaging/imaging.py

```diff
@@ -140,12 +138,20 @@ class MaskType(dj.Lookup):
 @schema
 class ProcessingTask(dj.Manual):
     definition = """
-    -> scan.Scan
+    -> Session
     -> ProcessingParamSet
-    ---
+    scan_ids: varchar(16)  # list of the scan_id associated with this ProcessingTask (delimited by '_' for example)
+    ---    
     task_mode='load': enum('load', 'trigger')  # 'load': load computed analysis results, 'trigger': trigger computation
+    processing_output_dir: varchar(255)        #  output directory relative to root data directory
     """
 
+    class MemberScan(dj.Part):
```

Comment:
needs a comment explaining.

Reply:
or you may rename this class to be more self explanatory e.g. `ScanInSequence` or `ScanInGroup` or `ScanToProcess`.